backend/controllers/subscriptionController.js

```diff
@@ -58,4 +73,23 @@ const getSubscription = async (req, res) => {
   }
 };
 
-module.exports = { postSubscription, getSubscription, getPayment };
+
+const subscriptionPlans = (req, res) => {
+  const { platform, followers } = req.body;
+
+  if (!platform || !followers) {
+      return res.status(400).json({ error: 'Platform and followers are required' });
+  }
+
+  const charges = calculateCharges(followers);
+
+  return res.json({
+      platform,
+      followers,
+      charges,
```

Comment:
it should only return the list of subscriptions containing price and duration(quaterly, halfYearly, Annualy)

Reply:
Ok